ui/js/main.js

```diff
@@ -8,6 +8,7 @@ import store from "store.js";
 import SplashScreen from "component/splash";
 import { doChangePath, doNavigate, doDaemonReady } from "actions/app";
 import { toQueryString } from "util/query_params";
+import setTheme from "util/setTheme";
```

Comment:
This also needs to be refactored. This needs to be done react/redux style, and should fit into existing settings selectors/reducers/actions.